Formula/e/erlfmt.rb

```diff
@@ -0,0 +1,20 @@
+class Erlfmt < Formula
+  desc "Automated code formatter for Erlang"
+  homepage "https://github.com/WhatsApp/erlfmt"
+  url "https://github.com/WhatsApp/erlfmt/archive/refs/tags/v1.6.1.tar.gz"
+  sha256 "198f33ff305806d4cee5113884364d8874912f3ad0ec3d0aa2ca243cfadc44b6"
+  license "Apache-2.0"
+
+  depends_on "rebar3" => :build
+  depends_on "erlang"
+
+  def install
+    system "rebar3", "as", "release", "escriptize"
+    bin.install "_build/release/bin/erlfmt"
+  end
+
+  test do
+    assert_equal "f(X) -> X * 10.\n",
```

Comment:
Can you add a version check test?
```suggestion
    assert_match version.to_s, shell_output("#{bin}/erlfmt --version")
    assert_equal "f(X) -> X * 10.\n",
```

Reply:
I can, but the Formula Cookbook I followed said ["(despite their widespread use) foo --version and foo --help are bad tests"](https://docs.brew.sh/Formula-Cookbook#add-a-test-to-the-formula:~:text=(despite%20their%20widespread%20use)%20foo%20%2D%2Dversion%20and%20foo%20%2D%2Dhelp%20are%20bad%20tests) ??

Reply:
When you test it only, that is bad but when there are others, it will be fine.